EpsilonAuxBMS/Inc/AuxBMS.h

```diff
@@ -0,0 +1,92 @@
+#pragma once
```

Comment:
This file is included by both `AuxBMS.h` & `AuxCAN.h`, that's kinda weird. Let's just have two different AuxBMS & AuxCAN headers.